.github/workflows/cts.yml

```diff
@@ -1,11 +1,16 @@
 name: CTS
 
 on:
+  push:
+    branches-ignore: [
+        # We don't need to run on renovate PRs.
+        "renovate/**",
+        # This is the branch the merge queue creates.
+        "gh-readonly-queue/**",
+      ]
+    tags: [v0.*]
```

Comment:
`v0.*` won't match our current releases, right? I assume that is the intent here.